src/main/webapp/cal.html

```diff
@@ -5,13 +5,27 @@
         <title>Welcome Page</title>
         <link rel="stylesheet" href="style.css">
         <script src="script.js"></script>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cinzel">
-        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Parisienne">
     </head>
     <body onload="insertCalendar()">
-      <div id="caldiv">
-      </div>
-      <button id="createCalendarButton" onclick="createCalendar()">Create a family calendar</button>
-      <button id="deleteCalendarButton" onclick="deleteCalendar()">Delete family calendar</button>
+        <div class="colorstrip" class="top-container">
```

Comment:
indent -1

Reply:
Everything else seems fine

Reply:
I think this is fine because the colorstrip is inside the body and this is the formatting in the other files